libs/@hashintel/design-system/src/text-field.tsx

```diff
@@ -57,6 +57,7 @@ export const TextField: FunctionComponent<TextFieldProps> = forwardRef(
           ...(Array.isArray(sx) ? sx : [sx]),
         ]}
         variant={variant}
+        minRows={3}
```

Comment:
This was really annoying to debug, there are several props in our `TextField` component which cannot be overridden. Should I move `{...textFieldProps}` to the bottom of this so that we can override anything we want by default @CiaranMn ?

Reply:
Yes please

Reply:
Done in https://github.com/hashintel/hash/pull/3694/commits/a04a6bf6184f8fe3e80d95b3fb8949b8c65eb9ae